File: C/src/Colleen.c
#include<stdio.h>

/*
	outside comment
*/
void lol() {}

int main()
{
	/*
		inside comment
	*/
	char *s = "#include<stdio.h>%1$c%1$c/*%1$c%2$coutside comment%1$c*/%1$cvoid lol() {}%1$c%1$cint main()%1$c{%1$c%2$c/*%1$c%2$c%2$cinside comment%1$c%2$c*/%1$c%2$cchar *s = %3$c%4$s%3$c;%1$c%2$cprintf(s, 10, 9, 34, s);%1$c%2$clol();%1$c}%1$c";
	printf(s, 10, 9, 34, s);
	lol();
}

File: Ruby/Colleen.rb
# comment 1
def lol; 0; end

def main
  # comment 2
  s = "# comment 1\ndef lol; 0; end\n\ndef main\n  # comment 2\n  s = %s\n  printf(s, s.inspect)\n  lol\nend\n\nmain\n"
  printf(s, s.inspect)
  lol
end

main
